authors/apps/articles/views.py

```diff
@@ -330,7 +363,7 @@ def delete(self, request, article_id):
 
         user_data = JWTAuthentication().authenticate(request)
 
-        serializer_data["user_id"] = user_data[1]
+        serializer_data["author"] = user_data[1]
```

Comment:
Did you update this variable in all necessary files

Reply:
Yes, I did.